src/Storage/Storage.Test/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,40 @@
+Invoke-LiveTestScenario -Name "Creates a Storage account" -Description "Test create storage account" -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $name = New-LiveTestResourceName
+    $vaultLocation = "westus"
+
+    $sa = Get-AzStorageAccount -ResourceGroupName $rgName -Name $name  
+    if ($null -eq $sa) {
+        $actual = New-AzStorageAccount  -ResourceGroupName $rgName -Name $name  -Location $vaultLocation -SkuName Standard_GRS 
+    }
+
+    Assert-AreEqual $name $actual.Name
+    Assert-AreEqual $rgName $actual.ResourceGroupName
+    Assert-AreEqual $vaultLocation $actual.Location
+    Assert-AreEqual "Standard_GRS" $actual.SkuName
+    Assert-AreEqual $false $actual.EnabledForDeployment
+    Assert-True { $actual.AllowBlobPublicAccess } "By default AllowBlobPublicAccess should be true"
+    Assert-Null $actual.AllowSharedKeyAccess "By default AllowSharedKeyAccess should be null"
+    Assert-False { $actual.EnableHttpsTrafficOnly } "By default EnableHttpsTrafficOnly should be false"
+}
+
+Invoke-LiveTestScenario -Name "Removes a Storage account" -Description "Test removes a Storage account from Azure." -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $name = New-LiveTestResourceName
+    $vaultLocation = "westus"
```

Comment:
Please rename the $vaultLocation to $location and update below invocation accordingly.